databricks/koalas/base.py

```diff
@@ -385,6 +385,9 @@ def astype(self, dtype):
 
         >>> ser.rename("a").to_frame().set_index("a").index.astype('int64')
         Int64Index([1, 2], dtype='int64', name='a')
+
+        >>> ser.astype('int63')
+        ValueError: Type int63 not understood
```

Comment:
For such test, we should add it under `koalas/tests/...` rather than doctest because it becomes the part of documentation.

Reply:
Thanks for comment. i'll fix and commit again

Reply:
i've fixed them!